src/modules/dashboard/components/Dashboard/UserTasks.css

```diff
@@ -7,3 +7,13 @@
 .coloniesContainer {
   margin-top: 40px;
 }
+
+.filter {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.taskList {
+  display: flex;
+  max-height: calc(100vh - 450px);
```

Comment:
This one produces the following result:

![Task List 2](https://user-images.githubusercontent.com/3052635/65904284-c3666780-e383-11e9-8eae-df9559d9a2e2.gif)